pkg/odo/cli/component/common_push.go

```diff
@@ -330,3 +332,75 @@ func filterIgnores(filesChanged, filesDeleted, absIgnoreRules []string) (filesCh
 	}
 	return filesChangedFiltered, filesDeletedFiltered
 }
+
+// retrieveKubernetesDefaultNamespace tries to retrieve the current active namespace
+// to set as a default namespace
+func retrieveKubernetesDefaultNamespace() (string, error) {
```

Comment:
Maybe some simple name like `GetCurrentNamespace` like we have in the occlient package?

Reply:
the reasoning for the naming is because it's just a private function

Reply:
But I think we can export this function. It might be useful at many other places later on.